.github/workflows/scenario-test.yml

```diff
@@ -125,9 +126,9 @@ jobs:
           sed -i "s/instance_name = \"my-cf-org-name\"/instance_name = \"cf-instance-$(($RANDOM % 9 + 1))\"/" released/discovery_center/mission_4327/setup_subaccount_cf/main.tf
           if [ "${{ matrix.usecase.use_custom_idp }}" = "true" ]
           then 
-           else.folder }} apply -var-file="${{ matrix.usecase.var_file }}" -var cf_org_user=${{ secrets.GH_ACTION_USER_1 }} -var cf_space_manager=${{ secrets.GH_ACTION_USER_1 }} -var "btp_user=${BTP_USERNAME}" -var "btp_pass=${BTP_PASSWORD}" -auto-approve -no-color 
+           else.folder }} apply -var-file="${{ matrix.usecase.var_file }}" -var='cf_org_user=["${{ secrets.GH_ACTION_USER_1 }},${{ secrets.GH_ACTION_USER_2 }}"]' -var='cf_space_manager=["${{ secrets.GH_ACTION_USER_1 }},${{ secrets.GH_ACTION_USER_2 }}"]' -var "btp_user=${BTP_USERNAME}" -var "btp_pass=${BTP_PASSWORD}" -auto-approve -no-color 
```

Comment:
should this really be `else.folder`? Shouldn't this be `terraform -chdir=${{ matrix.usecase.folder }}`